packages/kit/src/runtime/server/validate-headers.js

```diff
@@ -0,0 +1,59 @@
+/** @type {Set<string>} */
+const VALID_CACHE_CONTROL_DIRECTIVES = new Set([
+	'max-age',
+	'public',
+	'private',
+	'no-cache',
+	'no-store',
+	'must-revalidate',
+	'proxy-revalidate',
+	's-maxage',
+	'immutable',
+	'stale-while-revalidate',
+	'stale-if-error',
+	'no-transform',
+	'only-if-cached',
+	'max-stale',
+	'min-fresh'
+]);
+
+const CONTENT_TYPE_PATTERN = /^[a-z]+\/[-+.\w]+$/i;
```

Comment:
This won't work as-is: `Content-Type` isn't just `type/subtype`, it's `type/subtype[;parameter=value]` -- for example, `Content-Type: text/html; charset=UTF-8`. There are other parameters that you can provide as well. We could opt out of checking these for validity by splitting the value on `;` and only taking the first part? I'm not sure the spec is specific enough on the parameters to make validation useful.